src/components/dropdown/dropdown.js

```diff
@@ -74,6 +77,7 @@ export default class Dropdown extends PureComponent {
 
   select = value => () => {
     if (this.props.onChange) {
+      this.setState({ showCustomLabel: false });
```

Comment:
Instead of using the state, you can use know if you need to use the label or the current value on the render function, this way you avoid the need of an extra state

Reply:
How do you do it? `current` will not be null as long as there are dropdown items since it will be the label of either the first or the clicked dropdown item, but I want the custom label to show until any dropdown item is clicked.

Reply:
You can change the line [104](https://github.com/couds/react-bulma-components/pull/140/files#diff-197080e688816258787520857120d94aR104) to something like
```
if (child.type === DropdownItem && ((i === 0 && !label ) || child.props.value === value))
```
and set `current = label` instead of `null`

With this the label will be used if no value match and the first value will be uset if no label

Reply:
Will try. Thanks.

Reply:
@couds removed `showCustomLabel` state; updated the behavior of custom label (original comment updated)